sdk/python/feast/inference.py

```diff
@@ -99,6 +99,10 @@ def update_feature_views_with_inferred_features_and_entities(
     other columns except designated timestamp columns are considered to be feature columns. If
     the feature view already has features, feature inference is skipped.
 
+    Note that this inference logic currently does not take transformations into account. For
```

Comment:
Probably clarify transformations and aggregations.

Reply:
wdym?

Reply:
Like clarify that stream feature views doesn't take into account udf or aggregations(transformations is a little ambiguous)

Reply:
addressed